include/maxscale/config_common.hh

```diff
@@ -47,6 +47,16 @@ using DiskSpaceLimits = std::unordered_map<std::string, int32_t>;
 #define DEFAULT_ADMIN_HTTP_PORT 8989
 #define DEFAULT_ADMIN_HOST      "127.0.0.1"
 
+/** SSL versions for the REST API */
+enum admin_ssl_version_t
```

Comment:
Use the `maxbase::ssl_version::Version` enum from `<maxbase/ssl.hh>`, this way all the TLS version related code uses the same enumeration values.